index.html

```diff
@@ -29,31 +29,18 @@
         </nav>
         </header>
         <section id="headline-section">
-            <h4 class="headline-title"> Hi there!👋🏼
-                I'm JD 🤭</h4>
+            <h1 class="headline-title"> Hi there!👋🏼 <br>
+                I'm JD.</h1>
+                <h2 class="headline-sub-title"> I'm a Micronout 👨🏼‍🚀💻</h2>
```

Comment:
- [Optional] Dear @JuanDiegoGuerra, please consider removing icons that are not part of the figma design (👋🏼👨🏼‍🚀💻). As front-end developers, we need to mimic the design and deliver something identical.

Reply:
Hello @TadesseDev, I just removed the icons. I know this is part of being professional but I think that sometimes your work should have some of your **spirit** or **personality**, cause that won't make you see less professional or serious and that also differences us from the computers hehe. I think this has a value for me and in some point you could remember the guy who tell you a joke in a job interview insteed the others that only went to do what they had to do. I really appreciate your feedback and I'm continue fixing the issues you pointed.

Reply:
I know right😊. But the thing is. we are considering ourselves as someone who have just received a template to work with. The test is all about delivering the giving template. Once you have completed the portfolio project you will have the freedom of customizing it to the point you like it anyhow. I'm sorry for giving you a hard time brother @JuanDiegoGuerra  😊. But I'm sure you are going to learn a lot as we all did.

Reply:
No problems mate @TadesseDev. I'm glad I'm reiciving good and helpful feedback! 💪🏼